README.md

```diff
@@ -35,10 +35,10 @@ Runnable runner = (Runnable) aClass.newInstance();
 runner.run();
 ````
      
-I suggest making you class implement a KnownInterface of your choice as this will allow you to call/manipulate instances of you generated class.
+I suggest making your class implement a Known Interface of your choice as this will allow you to call/manipulate instances of you generated class.
```

Comment:
KnownInterface is deliberately one word to emphasize the camel-case interface name, can you undo this pls

Reply:
Oh sorry, changed it.